content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -225,66 +291,77 @@ Admin URL: https://app.netlify.com/sites/YOUR_SITE_NAME
+   ![Netlify environment variables](images/500-07-deploying-to-netlify-environment-variables-settings.png)
 5. Click **Save**.
 
-## 9. Deploy the app
+## 9. Test the deployed app
 
-Your project is now ready for deployment
+You can now test the deployed application.
 
-You can now use `git push` to push the commit with the Prisma schema. This will trigger a netlify build.
+### Steps
 
-**Checkpoint:** Call the API status endpoint:
-
-```terminal
-curl https://YOUR_SITE_NAME.netlify.app/.netlify/functions/status
-```
-
-The call should return: `{"up":true}`
-
-## 10. Test the deployed app
-
-Open the site from the command line:
+Open the site with the Netlify CLI.
 
 ```terminal
 netlify open:site
 ```
 
-This will open the static frontend that you deployed.
-The URL should like this: `https://YOUR_SITE_NAME.netlify.app`.
+### Result
+
+Your default browser opens the static frontend that you deployed.
+The URL should look like this: `https://YOUR_SITE_NAME.netlify.app`.
 
-![deployed-screenshot](https://res.cloudinary.com/prismaio/image/upload/v1628765208/docs/RnaHMS3_urf3iu.png)
+![Netlify application with Prisma deployed](images/500-08-deploying-to-netlify-application-deployed.png)
 
 The four buttons allow you to make requests to the REST API and view the response:
 
-- **Check API status**: Will call the REST API status endpoint that returns `{"up":true}`. The implementation code is in `functions/index.js`
-- **Seed data**: Will delete all database records and load the database with test data `users`, `profiles`, and `posts`. Returns the created users. The implementation code is in `functions/seed.js`
-- **Load users with profiles**: Will load all `users` in the database with their related `profiles`. The implementation code is in `functions/getUsers.js`
-- **Load Posts**: Will load `posts` and their related `authors`. The implementation code is in `functions/getPosts.js`
+- **Check API status**. Calls the REST API status endpoint that returns `{"up":true}`. The implementation code is in `functions/index.js`
+- **Seed data**. Deletes all database records and loads the database with test data `users`, `profiles`, and `posts`. Returns the created users. The implementation code is in `functions/seed.js`
+- **Load users with profiles**: Loads all `users` in the database with their related `profiles`. The implementation code is in `functions/getUsers.js`
+- **Load Posts**: Loads `posts` and their related `authors`. The implementation code is in `functions/getPosts.js`
 
-For example, calling seed data should show the following:
+For example, when you slick Seed data, the application shows the following:
 
-![deployed-screenshot](https://res.cloudinary.com/prismaio/image/upload/v1628765199/docs/dOJUBsv_e6fx9m.png)
+![Netlify application with Prisma deployed - Call result](images/500-09-deploying-to-netlify-application-deployed-call-result.png)
 
 ## Summary
```

Comment:
I think this is missing a bit that informs the user that now every future commit to the repo will deploy automatically (for `main` branch) or create preview deployments (for PRs) because of the way the integration is set up. So no more need to use the CLI for anything.